frontend/src/modules/Environments/services/listAllEnvironmentConsumptionRoles.js

```diff
@@ -29,11 +29,6 @@ export const listAllEnvironmentConsumptionRoles = ({
           groupUri
           IAMRoleArn
           dataallManaged
-          managedPolicies {
```

Comment:
removing this from here as this is what was causes the gql to load slowly. Replacing this with an asynchronous way of fetching this same information

Reply:
If we are not going to use it in the frontend, we should also remove it from the backend gql type where we are fetching it. (`backend/dataall/core/environment/api/types.py:183`) Please ensure that there is no other place where we get or list ConsumptionRoles and use the managedPolicies field

Reply:
Let me check and remove it